v1/related.js

```diff
@@ -11,59 +11,98 @@ class Related {
         this._options = options;
     }
 
-    getPages(hyper, req) {
-        const rp = req.params;
-        const rh = Object.assign({}, req.headers);
-        // we don't store /page/related no need for cache-control
-        if (rh['cache-control']) {
-            delete rh['cache-control'];
-        }
+    _relatedPageRedirect(hyper, domain, title) {
+        const apiURI = new URI(`https://${domain}/w/api.php`);
```

Comment:
This still doesn't use the Action API wrapper but rather calls this URI directly which then makes an external call to the MW instance instead of a local call. This will also miss out on error handling provided by the Action API wrapper since the Action API returns 200 on errors.

To use the Action API wrapper you'd need to modify URI to something like:
```suggestion
        const apiURI = new URI([domain, 'sys', 'action', 'rawquery']);
```

Reply:
Thank you for the suggestion, pls check recent changes.